ext/src/ruby_api/params.rs

```diff
@@ -10,9 +10,6 @@ struct Param {
     index: u32,
     ty: ValType,
 }
-// Keep `Param` small so copying it to the stack is cheap, typically anything
-// less than 3usize is good
-assert_eq_size!(Param, [u64; 2]);
```

Comment:
`ValType` is not `Copy` anymore, so I don't think this assert is useful anymore, but let me know if there's anything that I'm missing here cc @ianks

Reply:
It may still have a small performance impact (we can benchmark it) but if that's the case we can come back to optimize it, it doesn't sound like a big deal?

Reply:
I don't think it's a big deal to be honest, but we can always come back later if this ends up being an issue.

Reply:
What's the new size of it? Can we assert on the new size instead to avoid accidentally bloating it?